ezcontent_theme.theme

```diff
@@ -67,3 +67,20 @@ function ezcontent_theme_preprocess_menu_local_action(&$variables) {
     '#url' => $link['url'],
   ];
 }
+
+function ezcontent_theme_preprocess_block(&$variables) {
+  $variables['site_name'] = \Drupal::config('system.site')->get('name');
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function ezcontent_theme_theme_suggestions_block_alter(&$suggestions, $variables) {
+  $content = $variables['elements']['content'];
+  $content_layout = $variables['elements']['content']['content'];
+  if (isset($content['#block_content']) && $content['#block_content'] instanceof \Drupal\block_content\BlockContentInterface) {
```

Comment:
Please declare the class with a use statement and then access it here via the class name only.